mods/ra/maps/allies-07/allies07.lua

```diff
@@ -158,13 +159,14 @@ ClearSubActivityCompleted = function()
 	greece.MarkCompletedObjective(ClearSubActivity)
 end
 
+SovietArmoredBattalionArrivesIn = UserInterface.Translate("soviet-armored-battalion-arrives-in")
 Tick = function()
 	ussr.Cash = 5000
 	badguy.Cash = 500
 
 	if StartTimer then
 		if ticked > 0 then
-			UserInterface.SetMissionText("Soviet armored battalion arrives in " .. Utils.FormatTime(ticked), TimerColor)
+			UserInterface.SetMissionText(SovietArmoredBattalionArrivesIn .. " " .. Utils.FormatTime(ticked), TimerColor)
```

Comment:
This isn't correct. Other languages may not want to put the number at the end, its place supposed to be configurable. The translators also need to be made aware of the numbers existance